pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,522 @@
+</Callout>
+
+</Steps>
+
+## Verifying Docker Installation
+
+To ensure that Docker has been successfully installed and is running on your system, follow the steps below:
+
+<Steps>
+{<h3>Open the Command Line or Terminal</h3>}
+
+- **Windows**: Press `Windows + R`, type `cmd`, and hit Enter.
+- **Mac**: Press `Command + Space`, type `terminal`, and hit Enter.
+- **Linux**: Depending on the distribution, you can use shortcuts like `Ctrl + Alt + T` or access the terminal from the application menu.
+
+{<h3>Verify Docker Version</h3>}
+
+Type the following command:
+
+```
+docker --version
+```
+
+This should display the version of Docker installed on your machine. For example:
+
+```
+Docker version 24.0.6, build ed223bc
+```
+
+{<h3>Check Docker Details</h3>}
+
+For more detailed information about the Docker installation, you can use:
+
+```
+docker info
+```
+
+This command provides a comprehensive overview of your Docker setup, including the number of containers running, paused, and stopped, as well as other system information.
+
+{<h3>Ensure Docker Daemon is Running</h3>}
+
+To check that the Docker daemon (background service) is running, execute:
+
+```
+docker system info
+```
+
+If Docker is running correctly, this command will return system-wide information. If not, you may receive an error indicating that Docker is not running.
+
+</Steps>
+
+### Troubleshooting
+
+If you encounter any errors or if Docker does not seem to be running:
+
+- Ensure that the Docker Desktop application (for Mac and Windows) is running.
+- On Linux, you might need to start the Docker service manually using a command like `sudo service docker start` or `sudo systemctl start docker`, depending on your Linux distribution.
+
+## Managing Docker Service on Windows using PowerShell
+
+On Windows, Docker typically runs as a service via Docker Desktop. While Docker
+Desktop provides a user-friendly interface for managing Docker, there are
+instances where you might want to control the Docker service directly from the
+command line using PowerShell.
+
+Here's how you can manage the Docker service on Windows using PowerShell:
+
+### Open PowerShell as an administrator
+
+- Right-click the Start button and select "Terminal (Admin)" or search
+  for "PowerShell" in the start menu, right-click "Windows PowerShell", and
+  select "Run as administrator".
+
+### Start the Docker Service
+
+If Docker isn't running and you want to start it, use the following command:
+
+```powershell
+net start com.docker.service
+```
+
+You should receive a message indicating that the Docker service has started successfully.
+
+### Stop the Docker Service
+
+To stop the Docker service, use the following command:
+
+```powershell
+net stop com.docker.service
+```
+
+You should receive a message indicating that the Docker service has stopped.
+
+### Check the Status of the Docker Service
```

Comment:
I think I would move this as a first paragraph, like
1. check if it's running or not
2. start it or stop it depending on what you need

Reply:
Moved.